classes/patterns/class-takeaction.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * TakeAction pattern class.
+ *
+ * @package P4GBKS
+ * @since 0.1
+ */
+
+namespace P4GBKS\Patterns;
+
+use P4GBKS\Patterns\Templates\Covers;
+
+/**
+ * Class TakeAction.
+ *
+ * @package P4GBKS\Patterns
+ */
+class TakeAction extends Block_Pattern {
+
+	/**
+	 * Returns the pattern name.
+	 */
+	public static function get_name(): string {
+		return 'p4/take-action-pattern-layout';
+	}
+
+	/**
+	 * Returns the pattern config.
+	 *
+	 * @param array $params Optional array of parameters for the config.
+	 */
+	public static function get_config( $params = [] ): array {
+		$classname = self::get_classname();
+
+		return [
+			'title'      => __( 'Take Action', 'planet4-blocks-backend' ),
```

Comment:
As like the block names we can disallow block pattern names translation. :) (if admin translate block pattern name then new editor/admin user may confuse the block pattern name on site and in handbook documentation)

Reply:
That's a good point! I've added a [commit](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/925/commits/a47ddeb304305ed567ffbc5ec0b577413830aaa2) removing translations not only from this pattern, but from all of the existing ones